wrangler.jsonc

```diff
@@ -44,7 +44,8 @@
     ],
 	"vars": {
 		"HONEYCOMB_DATASET": "",
-		"HONEYCOMB_API_KEY": ""
+		"HONEYCOMB_API_KEY": "",
+		"HOST_NAME": "https://agent.thoughtspot.app"
```

Comment:
We have created aliases for this, and might also change this in the future. Lets not hardcode this. This should be available in the request payload headers.

Reply:
Yeah, added it as part of props, let me know if I need to change it.